src/SourceBuild/content/prep.sh

```diff
@@ -46,6 +48,9 @@ while :; do
       print_help
       exit 0
       ;;
+    --no-binaries)
+      keepBinaries=false
```

Comment:
The name of the switch and behavior don't seem to align.  I literal reading of this switch implies to me that the binaries will be removed.

Reply:
> I literal reading of this switch implies to me that the binaries will be removed.

This was my intention with the switch. It would be passed when binaries should be removed. Would something like `--remove-binaries` work better?

Reply:
Removing binaries should be the default behavior.  The option name should convey the default behavior.

Reply:
Used `keep-binaries`. Specifying this switch will not remove the binaries. The default script default behavior now is to remove the binaries. https://github.com/dotnet/installer/pull/18541/files#diff-a1d21cbd7f180cea469d4fb634d555679ca532f21d903b2b5df311f6a051aa6aR13